index.html

```diff
@@ -2,9 +2,118 @@
+          <li class="catalog-item"><a href="#"> Фитнес-браслеты</li>
+          <li class="catalog-item"><a href="#"> Умные часы</li>
+          <li class="catalog-item"><a href="#"> Квадрокоптеры</a></li>
+          </ul>
+          </section>
+          <section class="delivery">
+            <div>
+            <h2 class="delivery"">Доставка</h2>
+            <p class="delivery-text">Мы с удовольствием доставим ваш товар прямо к вашему подъезду совершенно бесплатно!Ведь мы неплохо заработаем,поднимая его на ваш этаж.</p>
+            <ul class="delivery-advantages">
+              <li class="delivery-item"> <a href="#"> Доставка </a></li>
+              <li class="delivery-item"> <a href="#"> Гарантия </a></li>
+              <li class="delivery-item"> <a href="#"> Кредит </a></li>
+            </ul>
+            </div>
+          </section>
+          <p class="delivery-text" href="Доставим редкий товар под заказ"></p>
+          <div>
+          <section>
+          <h3 class="about-us">О нас</h3>
+          <p class="about-text">Огромный выбор гаджетов не оставит равнодушным гика,который есть в каждом из нас.</p>
+          <p class="about-text">Мы доставим ваш товар в самые отдаленные уголки России транспортными компаниями:</p>
+          <ul class="about-transport-company">
+            <li class="trasport-item"> Меловые линии</li>
+              <li class="transport-item"> Сдик</li>
+              <li class="transport-item"> Токсберри</li>
+          </ul>
+        </div>
+          <a href="#">Подробнее о нас</a></p>
+          <div>
+          <h4 class="Contacts">Контакты</h4>
+          <p class="contacts-text">Вы можете забрать товар сами,заехав в наш офис.Заодно сможете проверить работоспособность покупки.</p>
+          <p class="internet-shop">Интернет магазин "девайс"</p>
+          <p class="adress">Санкт-Петербург,набережная реки Карповки,5,литера П.</p>
+          <p class="telephone-number"> <a href="+7(812)812-12-12"></a></p>
+          <p class="time-job"> Время работы</p>
+          <p class="time-job-day">пн-пт:с 10:00 до 20:00</p>
+          <p class="time-job-clock">сб-вс:с 10:00 до 19:00</p>
+          <a href="#" class="write-us">Напишите нам</a>
+          </section>
+        </div>
+          <section class="subscription">
+            <h5 class="subscription-list">Подпишитесь на рассылку</h5>
+            <p class="ubscription-text">Еженедельный дайджест с обзорами новинок,длительными тест-драйвами,распаковками,а также акции, спецпредложение и вот это вот все.</p>
+            <form>
+            <input type="email">
+            <a href="#" class="button" type="submit">Подписаться</a>
+            <p>Это бесплатно</p>
+          </form>
+          </section>
+      </main>
+      <footer class="page-footer">
+        <a class="logo" href=img alt="logo-device" src="#"></a>
+        <section class="footer-adress">
+          <p class="visually-hidden">Санкт-Петербург,набережная реки Карповки,5,литера П. </p>
```

Comment:
Ты ж видишь этот текст. А `visually-hidden` сделан для вещей, которые не видны, но используются в читалках.
```suggestion
          <p>Санкт-Петербург,набережная реки Карповки,5,литера П. </p>
```